server/server.go

```diff
@@ -141,6 +141,23 @@ func (s *Server) getPortForward(req *pb.PortForwardRequest) (*pb.PortForwardResp
 	return s.stream.streamServer.GetPortForward(req)
 }
 
+func (s *Server) wipeStorage() {
+	containers, err := s.Store().Containers()
+	if err != nil && !os.IsNotExist(errors.Cause(err)) {
+		logrus.Warnf("could not read containers and sandboxes: %v", err)
+	}
+	for i := range containers {
+		ctr := &containers[i]
+		_, err := s.StorageRuntimeServer().GetContainerMetadata(ctr.ID)
+		if err != nil {
+			continue
+		}
+		for _, name := range ctr.Names {
+			s.Store().DeleteContainer(name)
```

Comment:
Should be logging failures here.

Reply:
You are only deleting containers here, what about container images?

Reply:
@rhatdan I am unsure actually.
On the one hand, I can't seem to find a way to only wipe cri-o's image store like I did containers. Plus, having old images on the system won't break kubelet if they have updated, kubelet will just pull new ones.
On the other hand, those images won't be managed by kubelet and thus stick around unnecessarily. So in the interest of cleaning up after ourselves, I think it'd be better to delete them.

I am going to look into trying to clean only cri-o's images, but ultimately I think cleaning the images is the best option. cc @mrunalp